iModelCore/iModelPlatform/Tests/DgnProject/NonPublished/DgnStyles_Test.cpp

```diff
@@ -437,6 +437,7 @@ TEST_F(DgnLineStyleTest, InsertLineCodeComponentAsJson)
 
     ASSERT_TRUE(DbResult::BE_SQLITE_OK == m_db->SaveChanges());
     seedFilePath = m_db->GetDbFileName();
+    m_db->CloseDb();

```

Comment:
FYI: All of these were places where we inadvertently forgot to close the (writeable) db. We then open it again later in the test. That used to "succeed" because the first writeable connection saved its changes and therefore did not have a txn open, so the second writer "stole" its lock. Now, the second open fails.